site/src/modules/resources/AppLink/AppLink.tsx

```diff
@@ -61,12 +61,16 @@ export const AppLink: FC<AppLinkProps> = ({ app, workspace, agent }) => {
     app,
   );
 
+  // canClick is ONLY false when it's a subdomain app and the admin hasn't
+  // enabled wildcard access URL or the session token is being fetched.
+  //
+  // To avoid bugs in the healthcheck code locking users out of apps, we no
+  // longer block access to apps if they are unhealthy/initializing.
```

Comment:
If I'm reading this correctly, the UX will still broadly look the same, but the App Button will be clickable. Like, it'll still be easy to visually tell that something is wrong if healthchecking says it's unhealthy.  Right?

Reply:
Yep, there'll be a loading indicator for initializing apps or a circle with an exclamation mark in it. Hovering over will show a tooltip saying what the problem is (although currently we don't really store healthcheck failure reasons so it just says "Unhealthy")